Dockerfile-slim.template

```diff
@@ -16,6 +18,9 @@ RUN wget -qO- \
     https://github.com/uclouvain/openjpeg/archive/v${OPENJPEG_VERSION}.tar.gz | \
     tar -xzC $ROOTDIR/src/
 
+# Add repository to allow libproj-dev=7.1.0-1 installation
+RUN echo "deb  [ allow-insecure=yes ] http://http.us.debian.org/debian testing main non-free contrib" >> /etc/apt/sources.list
+
```

Comment:
This is dangerous because it switches the whole installation to the _"testing"_ distribution. When we install `libproj-dev`, it will pull all of its dependencies from _"testing"_ as well. Depending on `libproj-dev`'s dependency tree, this could brick parts of the installation.

I don't want to chance this because, for example, Raster Foundry uses this container image in production.

Have you considered downloading and installing this dependency from source, as we do above with GDAL and OpenJPEG?

Reply:
@rbreslow I don't buy that point, but I will get rid of the testing repo.

Reply:
Eh I just thought it is not a prod image and planned to use it for gt testing